ui/src/main/kotlin/edu/stanford/spezi/ui/ComposableContent.kt

```diff
@@ -0,0 +1,9 @@
+package edu.stanford.spezi.ui
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+
+interface ComposableContent {
```

Comment:
Consider adding 

```kotlin
@Composable
    fun Content() {
        Content(Modifier)
    }
```

in case of default modifiers, as default values are not supported in composable interface methods